Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Use an official Node.js runtime as the base image
+FROM node:18
```

Comment:
Hi @abansal15, 
1) why you have added this dockerfile? Please remove it as it seems pushed mistakenly (if you are running your local server using this dockerfile, I would mention that Oppia can be setup using Docker as well -- [refer](https://github.com/oppia/oppia/wiki/Installing-Oppia-using-Docker), and the python setup is going to be removed soon).
2) it seems your lint checks are failing on this PR. please do not use `--no-verify` flag while pushing your changes because it loads up the CI unnecessarily.

Reply:
Ok @Shivkant-Chauhan  I will try again  and will raise the pr asap

Reply:
you can push the changes in your current PR itself.